src/Microsoft.Health.Fhir.CosmosDb/Features/Operations/Export/CosmosExportOrchestratorJob.cs

```diff
@@ -19,24 +22,77 @@ namespace Microsoft.Health.Fhir.CosmosDb.Features.Operations.Export
     [JobTypeId((int)JobType.ExportOrchestrator)]
     public class CosmosExportOrchestratorJob : IJob
     {
+        private const int DefaultNumberOfTimeRangesPerJob = 100;
```

Comment:
Is there a better place to get this from? If I remember correctly it is a setting shared by all parallel async jobs in Cosmos, not just export.

Reply:
I'm not sure - SQL export has the same pattern, so I copied it from there. 

I could add `CosmosDataStoreConfiguration` (where these values are stored) to the export orchestrator and pull the value from there instead. I wasn't sure if injecting a lower-level class with data store specific info into a higher-level class (Export Orchestrator) was a good pattern or not. Happy to change it if you don't see any issues. 😄